src/components/book.js

```diff
@@ -1,11 +1,39 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/prefer-stateless-function */
+
 import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Book extends React.Component {
+  render() {
+    const { book, removeBook } = this.props;
+    const { id, title, category } = book;
+
+    const handleRemoveBook = () => {
+      removeBook(id);
+    };
```

Comment:
This should be implemented in the `BooksList` component. Then you just pass `handleRemoveBook` to book as props.

Reply:
DONE